src/integration/common/footer.js

```diff
@@ -46,5 +46,29 @@ export default () => {
         });
       });
     });
+
+    describe('Paragraphs', () => {
+      const footerParagraphs = document.querySelectorAll('footer div p');
+
+      footerParagraphs.forEach(footerParagraph => {
+        const paragraphText = footerParagraph.textContent;
+        const paragraphUrl = footerParagraph.getAttribute('href');
+
+        it('should be in the document', () => {
+          expect(footerParagraph).toBeInTheDocument();
+        });
+
+        it('should contain text', () => {
+          expect(paragraphText).toBeTruthy();
+        });
+
+        it('should match text and url', () => {
+          expect({
+            text: paragraphText,
+            url: paragraphUrl,
+          }).toMatchSnapshot();
```

Comment:
Paragraphs won't have a url, so we can remove it, and just expect the paragraph text to match the snapshot. In fact, we can use the contents of the "I can see the footer copyright and external linking text" test above.

Reply:
Removed and have updated test